### DISCUSSION TITLE
jsDelivr CDN Support

### DISCUSSION THREAD

**Discussion by kaaax0815** [coder/code-server#3145] [General]
Could you put your releases inside the repo, so you can use a CDN like jsDelivr to access to releases?

**Comment by jsjoeio** [comment]
Hmm...it might help us to know a bit more about what problem you're trying to solve and how this would help? Would you mind elaborating?

**Reply by kaaax0815** [reply]
The problem is that github doesnt support ipv6 and i have a ipv6 only server

**Reply by kaaax0815** [reply]
And sometimes github tend to be really slow

**Reply by oxy** [reply]
It's unfortunately not something that's on the top of our priorities right now; I'd suggest downloading our release packages and uploading them to a server with IPv6 for personal use, and then using that to download - or you can use `scp` to copy files to your server from a local desktop instead of downloading directly on the server.
